### PR TITLE
Deskop version

### PR DESCRIPTION
**Pull request by why-a-en**
Implementation of the desktop version (tried to stick to the figma template as much as possible)

The main section
- Added a header containing logo and navigation items
- Added background color and image for the main element
- Social icons and descriptions are also adjusted

The work section
-Added new grid columns and rows for the section
- Started implementing from the header and the black bar
- Adjusted cards and background image to the right position

The about me section
- Description of the section are moved to the inner part of the section
- Background image is added and repositioned
- Skill set list items are also relocated and aligned

The contact form
-Header text is moved to the left of the section and adjusted to the right position
-All the inputs, textarea, and a button are nested inside a recently created flex container to be aligned

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -307,6 +282,7 @@ h4 {
   height: 48px;
   padding: 12px;
   text-align: center;
+  margin-top: -37px;
 }
 
 .project-button:hover {
```
Comment (TadesseDev) [inline styles.css:288]: - [ ] Dear @anbehindY. You have done an incredible job on the design. But for your buttons please follow the design guideline and use the appropriate type. 
### Figma button interactions
image

File: index.html [hunk 2]
```diff
@@ -1,290 +1,357 @@
...
+            <div class="tech-stack">
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+          </div>
+          <div id="gg" class="project-button">See Project</div>
         </div>
```
Comment (TadesseDev) [inline index.html:124]: - [ ] Dear @anbehindY, I think you haven't considered the `hover` effect on each card. Normally the description of each of these projects is on top of their image with a dark opacity. Once the user hovers over the project the overlaying text disappeared and the `See Project button becomes visible. Please implement this feature. 

### Before hover
image

### After Hover
image

File: index.html [hunk 3]
```diff
@@ -1,290 +1,357 @@
...
-                <input id="fullname" name="name" type="name" placeholder="Full name" maxlength="30" required />
+              <input
+                id="fullname"
+                name="name"
+                type="name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
             </label>
```
Comment (TadesseDev) [inline index.html:295]: - [ ] Dear @anbehindY, Your form seems to lose its layout now. When I switch to the mobile version your form is not the same as the Figma template. Please fix this styling issue. 

### Your form on `375px` mobile
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @anbehindY 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @anbehindY,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**